docs/guide/transactions.rst

```diff
@@ -137,3 +137,39 @@ As always, see the full list of possible arguments in the AltaPay documentation.
     }
 
     transaction = Callback.create_invoice_reservation(api=api, **parameters)
+
+Refunding Transactions
+++++++++++++++++++++++
+
+When a transaction has been captured you might want to refund it again, fully or partial. This could be if the customer wants to return the order. The process is very similar to capturing a transaction.
+
+See the AltaPay documentation for ``API/refundCapturedReservation`` for full details.
+
+.. code :: python
+
+    from altapay import Transaction
+
+    transaction = Transaction.find('TransactionID', api=api)
+    callback = transaction.refund()
+
+    if callback.result == 'Success':
+        # Refund was successful
+        pass
+    else:
+        raise Exception('Not able to refund')
+
+Partial refunds can be performed in the following way, which also shows how to supply an order line. Notice that not providing an ``amount`` will result in a full return of the transaction.
+
+        .. code :: python
+
+            from altapay import Transaction
+
+            transaction = Transaction.find('TransactionID', api=api)
+            respnse = transaction.capture(
+                orderLines=[{
+                    'description': 'Blue Shirt',
+                    'itemId': '12345',
+                    'quantity': 1.0,
+                    'unitPrice': 19.95
+                }],
+                amount=19.95)
```

Comment:
What if this amount is different from the sum of orderLines.unitPrice?

Reply:
The orderlines are not used for the actual refund.. But some payment methods (Klarna) may use them.